### ISSUE TITLE
Purchase Zenhub Licence for the team

### ISSUE THREAD

**Issue by spydmobile** [PSaaS-Developers/Project_issues#96]

**Comment by spydmobile** [comment]
I think the project should pick up the annual cost of 95$ per user for the exec and primary developer.
https://www.zenhub.com/pricing?utm_source=Dashboard&utm_medium=Link&utm_campaign=Pricing-Plans

**Comment by BadgerOnABike** [comment]
I agree as this would allow for full utility by exec and illustrate the standing maintenance costs of the project.

**Comment by spydmobile** [comment]
Again, it would be good to point out HOW this must be done. First, the account must be made on behalf of the principle owner of the gitgub part, so in this case its the part 2 since part 1 would be to purchase a github licence for the psaas-developers organization, first, then a zenhub licence for same allowing psaas-developers to attach zenhub to private AND open repos.

Currently @nealmcloughlin is the spending authority and @BadgerOnABike is the principle on the PSaaS-Developers account. So how do we get money from Neal to Brett so that Brett can do part 1? Also, this is not something we should "Wait" on. It should be Actioned ASAP IMO.

**Comment by BadgerOnABike** [comment]
Ownership is now @spydmobile @nealmcloughlin and myself. Thus any one of us can perform the action.

**Comment by nealmcloughlin** [comment]
I agree with purchasing a github license and zenhub account for the team. There are two options for how to proceed: (1) We get Rob to pay for these licenses and pay his HSS invoice through CIFFC. (2) Brett get's approval to pay for these licenses through CFS. This would be recognized as an annual CFS contribution to the project. AB and NWT have paid for these kind of license and hosting fees for years. I no longer have the support to do this in BC.

**Comment by spydmobile** [comment]
No, Sorry, Neal. Robert has no role in the Executive, involving him will only complicate matters worse than they are. so I disagree that this is an approach. Also, I would like to stop leveraging agencies paying bills please. We have money in the bank, I counter your suggestion offering that we need to establish an annual budget to pay our own predictable bills, and that the correct way to do this, is for us to establish a list of annual expenses, and start paying them from the project, not NT, not BC, not CFS and then chasing financial paperwork. If we dont start paying our own bills now, we never will and I Honestly hate that approach as it feels like laziness and passing the buck.

**Comment by BadgerOnABike** [comment]
Further, the plot thickens when we go to upgrade GitHub at least as it wants to license all team members and will not allow for a sub-selection, thus increasing the overall cost substantially.

**Comment by nealmcloughlin** [comment]
Don't disagree Franco. My suggestions were a way to move quickly. I will follow up with Anoosh to see if there is a mechanism for us to charge these costs directly to CIFFC. We may be able to charge these costs to a CIFFC credit card. If this option exists it will need to be facilitated by CIFFC as they are unlikely to give us their credit card number. We therefore need to be ready to with very explicit directions on what we need them to purchase. The only other option is for one of us to charge to our personal credit card and reclaim our costs by invoicing CIFFC like HSS does. I'm sending Anoosh an email this morning. Brett and Franco, please put together a costing sheet for what we need to purchase and for whom.

**Comment by spydmobile** [comment]
Ok I will make a couple new issue related to this and enfold this one into it.

**Comment by nealmcloughlin** [comment]
I heard back from Anoosh Shinnan at CIFFC. We have two options for paying for annual subscription fees such as GitHub and Zenhub:

1.) Pay for the fees with our own personal credit cards and submit an expense claim to CIFFC. Anoosh can provide the expense claim form.

2.) Coordinate payment with Anoosh and have her charge it to her CIFFC credit card.

**Comment by spydmobile** [comment]
> 1.) Pay for the fees with our own personal credit cards and submit an expense claim to CIFFC. Anoosh can provide the expense claim form.
> 
> 2.) Coordinate payment with Anoosh and have her charge it to her CIFFC credit card.

Thanks @nealmcloughlin
